packages/gafl-webapp-service/src/pages/contact/contact/__tests__/route.spec.js

```diff
@@ -80,30 +80,32 @@ describe('name > route', () => {
     )
 
     it.each([
-      ['Email', '12M', 'Email test@email.com', 'test@email.com'],
-      ['Text', '12M', 'Email', undefined],
-      ['Email', '8D', 'Email', 'test@email.com'],
-      ['Text', '8D', 'Email', undefined]
+      ['Email', '12M', false, 'Email test@email.com', 'test@email.com'],
+      ['Text', '12M', false, 'Email', undefined],
+      [undefined, undefined, true, 'Email', undefined],
+      ['Email', '8D', false, 'Email', 'test@email.com'],
+      ['Text', '8D', false, 'Email', undefined]
```

Comment:
This is getting pretty complicated and could do with being switched to a tagged template literal, e.g.
````
```
preferredMethodOfConfirmation | licenceLength | junior   | expected   | email
${'Email'}                    | ${'12M'}      | ${false} | ${'Email'} | ${'test@email.com'}
// etc
```
````

Also, the only email value returned is 'test@email.com', we could probably do with a different value for one of the two test cases